packages/teams-js/src/internal/messageObjects.ts

```diff
@@ -1,9 +1,28 @@
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export type BaseUUID = string;
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export type MessageUUID = BaseUUID;
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export type MessageID = number | MessageUUID;
+
 /**
  * @internal
  * Limited to Microsoft-internal use
  */
 export interface MessageRequest {
-  id?: number;
+  id?: MessageID;
+  uuid?: MessageUUID;
```

Comment:
Can you add a comment describing the difference between and proper usage of these two properties?